src/tests/actions/commentActions.test.js

```diff
@@ -0,0 +1,193 @@
+import expect from 'expect';
+import promiseMiddleware from 'redux-promise-middleware';
+import axiosconfig from '../../axiosConfig';
+import moxios from 'moxios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from '../../store/actions/commentsActions';
+import * as testData from '../testData';
+
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+
+
+describe('async actions', () => {
+  const middlewares = [promiseMiddleware()];
+
+  beforeEach(() => {
+    moxios.install(axiosconfig);
+  });
+  afterEach(() => {
+    moxios.uninstall(axiosconfig);
+  });
+  it('it dispatches FETCH_AN_ARTICLE on fetchAnArticle', () => {
+    const payload = testData.oneArticle;
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: payload,
+      });
+    });
+    const expectedActions = ['FETCH_AN_ARTICLE'];
+    // configure Mock store
+    const store = mockStore();
```

Comment:
I think it would be best to define the store in the global scope of this file so that you don't redefine it in every test